py/picca/fitter2/chi2.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import print_function
+import numpy as np
+import iminuit
+import time
+import h5py
+from numpy.linalg import cholesky
```

Comment:
same with scipy?

Reply:
agreed. The scipy.linalg should be slightly more performant.